src/constants/numbers.ts

```diff
@@ -22,4 +22,5 @@ export enum NumberSign {
 
 // Deposit/Withdraw
 export const MAX_CCTP_TRANSFER_AMOUNT = 1_000_000;
+export const MIN_CCTP_TRANSFER_AMOUNT = 10;
```

Comment:
Would these two variables be better in shared configs so mobile can leverage?

Reply:
max price impact may also be helpful

Reply:
yes will follow up to have the validation be in abacus!